### DISCUSSION TITLE
create a locked issue for announcements

### DISCUSSION THREAD

**Discussion by FranklinYu** [badges/shields#7520] [Ideas] [answered]
I’m using Shields for my public projects on GitHub, and I’d like to be notified when there is something deprecated (thus potential breaking change ahead). How about creating a locked issue (so that it won’t be spammed), then post news about breaking changes there? After that we can mention such a “mailing list” in ReadMe, so users can subscribe to it.

An example in the past is #5773.

**Comment by chris48s** [comment, marked as answer]
This is something we've mentioned before https://github.com/badges/shields/discussions/7017  and broadly agree.

Just to clarify: You say

> I’m using Shields for my public projects on GitHub

but then the issue you've linked to is a breaking change for self-hosting users. Are you interested in hearing about changes affecting users of shields.io or changes affecting users who host their own instance?

**Reply by FranklinYu** [reply]
I don’t host my own instance, so yes the example is inappropriate. I’ll follow #7017 instead. Thanks!
